OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -1,6 +1,9 @@
 $ErrorActionPreference = "Stop"
-$installStateFile = "$($env:SystemDrive)\Octopus\Octopus.Server.DSC.installstate"
 $octopusServerExePath = "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe"
+$script:instancecontext = ''  # a global to hold the name of the current instance's context
```

Comment:
Not sure why this global exists?

Reply:
Because the name of the instance we're currently working on gets passed around like a bong at a student house, and the scope was getting problematic. Actually, I think it's less important now the shared functions file is dot-sourced instead of loaded as a module.